src/components/Icon/Icon.mdx

```diff
@@ -0,0 +1,24 @@
+---
+name: Icon
+route: /components/icon
+menu: Components
+---
+
+import Icon from './Icon'
+import { Playground, PropsTable } from 'docz'
+
+# Icon
+
+## Examples
+
+<Playground>
+  <Icon name="home" />
+</Playground>
+
+<Playground>
+  <Icon name="thumbs-up" size={48} strokeWidth={1} color="#0071ce" />
```

Comment:
Can we not import this color from constants?

Reply:
Good catch! Now that we're using `docz` and `mdx` we can import the constants directly into markdown 🎉 That was another limitation of `react-styleguidist`